state/endpoint_bindings.go

```diff
@@ -433,6 +442,20 @@ func (b *Bindings) validateForCharm(charmMeta *charm.Meta) error {
 	return nil
 }
 
+// AllEndpointBindingsDocs returns all endpoint binding documents.
+func (st *State) AllEndpointBindingsDocs() ([]endpointBindingsDoc, error) {
```

Comment:
In _state/model.go_ there is `Model.AllEndpointBindings`, which we can use instead of adding this.

Reply:
ahh great I just couldn't find it, thanks!